cmd/buildah/commit.go

```diff
@@ -108,6 +109,7 @@ func commitListFlagSet(cmd *cobra.Command, opts *commitInputOptions) {
 		panic(fmt.Sprintf("error marking reference-time as hidden: %v", err))
 	}
 
+	flags.BoolVar(&opts.addHistory, "add-history", true, "add history for build layers (default true)")
```

Comment:
Can you make this disableHistory for consistency or stick to add-history throughout.

Reply:
Made this --add-history throughout, since we discussed this before here: https://github.com/containers/buildah/issues/3513#issuecomment-975899755